release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -47,6 +47,12 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-12-installation-and-upgrade"]
 === Installation and upgrade
 
+[id="ocp-4-12-gcp-azure-serial-console-logs"]
+==== Troubleshooting bootstrap failures during installation on GCP and Azure
+The installer now gathers serial console logs from the bootstrap and control plane hosts on GCP and Azure. This log data is added to the standard bootstrap log bundle. 
```

Comment:
Should it be "the installation program"? I feel like I've seen guidelines for that somewhere.

Reply:
I'm not sure about that - I didn't see anything in the contribution guidelines, and we refer to "the installer" and "the OpenShift installer" in a few places (including 4.11 release notes).